### DISCUSSION TITLE
How to disable Preview Deployments

### DISCUSSION THREAD

**Discussion by bartoszgolebiowski** [vercel/vercel#5878] [Help] [answered]
How to disable Preview Deployments

**Comment by tawaliou** [comment, marked as answer]
Just a question if I can, why you want to disable it ? If you want to avoid Search engines robots to index your preview, It is unnecessary. Vercel do it automatically by set X-Robots-Tag to noindex if **You are not using custom domains** like thy say [here](https://vercel.com/docs/platform/deployments). If not..

**Reply by dangdennis** [reply]
A use-case is that we at canopyservicing.com have multiple projects that all share the same repo. So a new preview will kick out for X projects regardless. We create a single project per each because we want more sets of env vars that's not just tied to git branches. But given that they're all identical, it'd be best for us to just have one preview for a single project.

**Reply by leerob** [reply]
You can use the "Ignored Build Step" to achieve this:

- https://vercel.com/blog/monorepos
- https://vercel.com/docs/platform/projects#ignored-build-step

**Reply by cheapsteak** [reply]
Ignore Build Step doesn't quite work
Branches and PRs created by other teammembers trigger failing builds since they're not in the project  

(we're exploring vercel, not quite ready to commit to migrating away from netlify yet, ended up disconnecting the repo for now so teammates don't all end up with red ❌ s on their PRs)

**Reply by leerob** [reply]
Hey @cheapsteak - are your teammates part of your Vercel team? I'm trying to understand if this is from the ignored build step, or if it's because you'll want to invite them to join your team.

**Reply by cheapsteak** [reply]
No they are not, it was just me on the team trial

I was hoping we could set up a parallel instance on vercel and gradually migrate all the netlify-specific parts, so when time comes to make a decision we could say it's already working in both places

Was not expecting failing status checks to show up on everyone's PRs as soon as I connected the repo and not being able to disable it without disconnecting the repo 😅

**Reply by leerob** [reply]
Sorry about that - rather than using the built-in integration during the transition to Vercel, you could deploy directly with a GitHub Action.

https://vercel.com/support/articles/how-can-i-use-github-actions-with-vercel

**Reply by kvedantmahajan** [reply]
@leerob I am deploying Vercel via GH actions but the problem is Vercel webhook still manages to mention "Ignored Deplyoment" leading to a failed status check on a protected branch. I believe if it is ignored, it should be considered successful status status and not a failure. It is a bug, IMO for ignored build step. 

<img width="1217" alt="Screenshot 2022-10-18 at 1 18 16 PM" src="https://user-images.githubusercontent.com/11805936/196368935-fb12535c-5da3-49de-bba4-d223d049d039.png">
<img width="972" alt="Screenshot 2022-10-18 at 1 20 31 PM" src="https://user-images.githubusercontent.com/11805936/196369530-c3e54071-345b-41fc-92aa-e2fe02043a7b.png">

**Reply by swarnava** [reply]
Hi @kushalmahajan I have noticed that you have already raised a ticket on Support. We will continue conversation there.
